backend/app/views/backend/publications/_form.html.slim

```diff
@@ -1,16 +1,16 @@
-== render 'backend/shared/form/header', form: form, title_placeholder: 'New Publication'
+= render 'backend/shared/form/header', form: form, title_placeholder: 'New Publication'
```

Comment:
Thanks for the changes! But as with the "News article title", I'd also do the same for Publications and Activities. For consistency!